### PR TITLE
Prepare relayer

### PR DESCRIPTION
**Pull request by Lbqds**

### CODE REVIEW

**Comment by h0ngcha0** [comment]
Q: Where is the code that does the actual redeem?

Lets deploy this and test it on testnet v2, I can help with that.

**Comment by Lbqds** [comment]
> Q: Where is the code that does the actual redeem?
> 
> Lets deploy this and test it on testnet v2, I can help with that.

I'm testing on the devnet now, I'll add a relayer.yml.template to the bridge infra repo.

File: relayer-engine/src/storage/redis-storage.ts [hunk 1]
```diff
@@ -154,7 +154,7 @@ export class RedisStorage implements Storage {
 
   async onTxSubmitted(vaaId: string, txId: string): Promise<void> {
     const txKey = `${vaaId}:txId`.toLowerCase()
-    await this.redis.set(txKey, txId)
+    await this.redis.set(txKey, txId, 'EX', 30 * 24 * 3600)
```
Comment (polarker) [inline relayer-engine/src/storage/redis-storage.ts:157]: Do we need 30 days cache here?

**Review by polarker — APPROVED** [review]
LGTM

File: relayer-engine/src/storage/redis-storage.ts (hunk 1 above)
Comment (Lbqds) [reply relayer-engine/src/storage/redis-storage.ts:157]: The purpose of storing the tx id is to be able to return it to the bridge ui. The assumption here is that the user will execute a redemption within 30 days, but seems that 30 days might be a bit long. How about one week?

Comment (polarker) [reply relayer-engine/src/storage/redis-storage.ts:157]: Yes, I think one week is enough

**Review by h0ngcha0 — APPROVED** [review]
LGTM :+1:

**Pull request merged**